test/integration/src/test/java/io/pravega/test/integration/KVTableWithStreamTest.java

```diff
@@ -0,0 +1,769 @@
+            valueArray1[i] = getValue();
+        }
+
+        // Adding 10 new entries to the Table only if they are not already present.
+        for (int i = 0; i < keyArray.length; i++) {
+            entryMap1.put(keyArray[i], valueArray1[i]);
+        }
+
+        List<Version> versionList1 = newKeyValueTable.putAll(keyFamily, entryMap1.entrySet()).join();
+        log.info("Version list1: {}", versionList1);
+        assertNotNull("Version list1 should not be empty", versionList1);
+        assertEquals("Version list1 size should be same as entry list1 size",
+                versionList1.size(), entryMap1.size());
+
+        List<Integer> keyList = new ArrayList<>();
+        for (int i = 0; i < keyArray.length; i++) {
+            keyList.add(keyArray[i]);
+        }
+        log.info("Key list: {}", keyList);
+        List<TableEntry<Integer, String>> getEntryList1 = newKeyValueTable.getAll(keyFamily, keyList).join();
+        log.info("Get Entry List1: {}", getEntryList1);
+        assertNotNull("Get Entry List1 should not be empty", getEntryList1);
+        assertEquals("Get Entry List1 size should be same as get entry list1 size",
+                getEntryList1.size(), entryMap1.size());
+        List<Integer> entryKeyList = new ArrayList<>();
+        for (Integer key: entryMap1.keySet()) {
+            entryKeyList.add(key);
+        }
+        log.info("entryKeyList: {}", entryKeyList);
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("Corresponding key for getEntryList should be as inserted",
+                    keyList.get(i), getEntryList1.get(i).getKey().getKey());
+            assertEquals("Corresponding key for entryList should be as inserted",
+                    entryKeyList.get(i), getEntryList1.get(i).getKey().getKey());
+        }
+
+        // Comparing each & every entry values of both entrylist and getEntryList.
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("All string values of both entryList1 and getEntryList1 should be equal",
+                    entryMap1.get(entryKeyList.get(i)), getEntryList1.get(i).getValue());
+        }
+        log.info("KVPair operations succeeds over KVTable");
+
+        // seal the stream.
+        CompletableFuture<Boolean> sealStreamStatus = controller.sealStream(newScope, streamName);
+        log.info("Sealing stream {}", streamName);
+        assertTrue(sealStreamStatus.get());
+        // delete the stream.
+        CompletableFuture<Boolean> deleteStreamStatus = controller.deleteStream(newScope, streamName);
+        log.info("Deleting stream {}", streamName);
+        assertTrue(deleteStreamStatus.get());
+        // delete the  scope.
+        CompletableFuture<Boolean> deleteScopeStatus = controller.deleteScope(newScope);
+        log.info("Deleting scope {}", newScope);
+        assertTrue(deleteScopeStatus.get());
+
+        log.info("Successfully completed creating Stream & KVTable with same " +
```

Comment:
This is useless. If the test completes, then you already know this. There is no point in logging it.

Reply:
okay.